docs/tutorials/offpolicy-tutorial.md

```diff
@@ -149,10 +151,10 @@ For the experiment configuration, we'll build on top of an existing
 [base BabyAI GoToLocal Experiment Config](/api/projects/babyai_baselines/experiments/go_to_local/base/#basebabyaigotolocalexperimentconfig).
 The complete `ExperimentConfig` file for off-policy training is
 [here](/api/projects/tutorials/babyai_go_to_local_bc_offpolicy/#bcoffpolicybabyaigotolocalexperimentconfig), but let's
-focus on the most relevant aspects to enable off-policy training. 
-Off-policy training involves instantiating
-[OffPolicyPipelineComponents](/api/utils/experiment_utils/#offpolicypipelinecomponent)
-and passing them to the current pipeline stage.
+focus on the most relevant aspect to enable this type of training: 
+We just need to instantiate a `TrainingPipeline` with an
+[OffPolicyPipelineComponent](/api/utils/experiment_utils/#offpolicypipelinecomponent)
+where required in our `PipelineStage` defined in the `training_pipeline` method.
```

Comment:
I think there is an error here. I would change this sentence from "We just need to instantiate a `TrainingPipeline` with an..." to:

"providing a [OffPolicyPipelineComponent](/api/utils/experiment_utils/#offpolicypipelinecomponent) object as input to a `PipelineStage` when instantiating the `TrainingPipeline`."